client/payment-details/summary/index.tsx

```diff
@@ -171,6 +172,19 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 
 	const isFraudOutcomeReview = isOnHoldByFraudTools( charge, paymentIntent );
 
+	moment.updateLocale( 'en', {
```

Comment:
That looks much simpler now. Can we add a comment explaining that we inject the translated messages to align on used locales and not expand the bundle size by moment translations library?

Reply:
Thanks. I have added it here 
d023fe1